docs/src/modules/spring/pages/actions.adoc

```diff
@@ -0,0 +1,77 @@
+= Implementing Actions in Spring 
```

Comment:
I'm wondering what will be the best title here. 

Should we repeat `in Spring` each time? Assuming that users are already under the Spring SDK chapter? 

The event sourced one is saying "with Spring".

Reply:
I think it helps when the user parachuts into these pages directly. And I kept it mainly to be consistent with the Java/Scala version... However, I see now that some pages have it others not, which makes it weird. Maybe we should remove it and just rely on the user to understand the chapter it is in?

Reply:
I'm thinking what if we share the link to this page? They will have no previous navigation.  I'm fine either way. Do we have a quorum?

Reply:
I deleted it here FYI

Reply:
If users arrive directly on the page, they still have the Spring section selected on the left side menu. I'm ok to remove for now. 

We can gives us more time to think about it and eventually re-introduce it.

Reply:
👍